pom.xml

```diff
@@ -115,6 +115,10 @@
     <url>https://builds.apache.org/view/A-D/view/Accumulo/</url>
   </ciManagement>
   <properties>
+    <accumulo.anyClusterTests>org.apache.accumulo.test.categories.AnyClusterTest</accumulo.anyClusterTests>
+    <accumulo.integrationTestsExcludedGroups />
+    <accumulo.integrationTestsGroups>${accumulo.anyClusterTests},${accumulo.miniclusterTests}</accumulo.integrationTestsGroups>
```

Comment:
I'd suggest shortening these property names to `accumulo.it.groups` and `accumulo.it.excludeGroups` to more closely match the user property names in the failsafe plugin, but without removing the accumulo-failsafe-specific context you've added.